src/tests/Microsoft.PowerFx.STJ.Tests/TypeSystemTests/FormulaTypeSerializerSnapshotTests.cs

```diff
@@ -102,7 +103,8 @@ public void TypeSnapshotSimple(string testId, string type)
                 {
                     // Serialize types without accounting for any defined type names
                     new FormulaTypeJsonConverter(new DefinedTypeSymbolTable())
-                }
+                },
+                DefaultIgnoreCondition = System.Text.Json.Serialization.JsonIgnoreCondition.WhenWritingDefault
```

Comment:
>DefaultIgnoreCondition [](http://example.com/codeflow?start=16&length=22)

Why do we need this? (will it impact other clients)

Reply:
This was the only way I could think of for getting rid of conditional json ignore here: 
[https://github.com/microsoft/Power-Fx/blob/main/src/libraries/Microsoft.PowerFx.Core/Public/Types/UserProvidedTypeDefinitions/FormulaTypeSchema.cs#L118](https://github.com/microsoft/Power-Fx/blob/main/src/libraries/Microsoft.PowerFx.Core/Public/Types/UserProvidedTypeDefinitions/FormulaTypeSchema.cs#L118)
Any other suggestion?